apps/src/aiDifferentiation/AiDiffPredefinedPrompts.tsx

```diff
@@ -203,3 +205,48 @@ export const IMPROVE_THIS_CODE = {
   label: 'Improve this code',
   prompt: 'How can this student code be improved?',
 };
+
+export const SUGGESTED_PROMPTS_FOR_SELECTION: {
+  [selection: string]: {initialMessage: string; suggestedPrompts: ChatPrompt[]};
+} = {
+  plan: {
+    initialMessage: `Let's iterate together! What would you like to change? Below are some of the tasks I can help you with.`,
```

Comment:
Do we already have a way to add new strings to translations/TMS? Happy to make this translated string, but also remember we had issue with adding new keys to TMS recently